### PR TITLE
Building for windows

### PR DESCRIPTION
**Pull request by allanleal**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by allanleal** [comment]
conda-forge-admin, please rerender

**Comment by allanleal** [comment]
@conda-forge-admin, please rerender

**Comment by allanleal** [comment]
@conda-forge-admin, please rerender

**Comment by allanleal** [comment]
@conda-forge-admin, please rerender

**Comment by allanleal** [comment]
@conda-forge/core please check failed re-rendering.

File: recipe/conda_build_config.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+c_compiler:    # [win]
+  - vs2017     # [win]
```
Comment (marcelotrevisani) [inline recipe/conda_build_config.yaml]: I think you can remove the ``# [win]`` selectors and use
```yaml
c_compiler:
  - vs2017
cxx_compiler:
  - vs2017
target_platform:
  - win-64
```

Comment (allanleal) [reply recipe/conda_build_config.yaml]: I'll try. Obrigado Marcelo. ;)

**Comment by allanleal** [comment]
@conda-forge-admin, please rerender

**Comment by marcelotrevisani** [comment]
@conda-forge-admin , please rerender

**Comment by marcelotrevisani** [comment]
But I think it will not work... because there are some ``yaml`` files which are specifying the vs to be 2008 and 2015

File: recipe/conda_build_config.yaml (hunk 1 above)
Comment (allanleal) [reply recipe/conda_build_config.yaml]: I'll probably remove also the osx selector as well.

File: .ci_support/win_c_compilervs2008cxx_compilervs2008.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+c_compiler:
```
Comment (marcelotrevisani) [inline .ci_support/win_c_compilervs2008cxx_compilervs2008.yaml]: Try to delete ``.ci_support/win_c_compilervs2008cxx_compilervs2008.yaml`` and ``.ci_support/win_c_compilervs2015cxx_compilervs2015.yaml``
after that you should rerender your file

Comment (allanleal) [reply .ci_support/win_c_compilervs2008cxx_compilervs2008.yaml]: Ok. I'll do in a moment.

**Comment by marcelotrevisani** [comment]
@conda-forge-admin , please rerender

**Comment by allanleal** [comment]
@conda-forge-admin, please rerender

**Comment by mariusvniekerk** [comment]
Looks like the one from #17 works correctly.  Guessing we want to close this one?

**Comment by allanleal** [comment]
@mariusvniekerk Yes, thanks to @marcelotrevisani for fixing this win issue!

**Pull request closed without merging**